cubedash/generate.py

```diff
@@ -142,10 +154,12 @@ def cli(
     force_concurrently: bool,
     verbose: bool,
     init_database: bool,
+    force_refresh: bool
 ):
     """
     Generate summary files for the given products
     """
+    _LOG.info("Running process to refresh")
```

Comment:
I'd change these human-friendly messages to `echo()` if we want them.

We're using `echo()` in this file for human cli output, and the [structlog](http://www.structlog.org) logger for machine-readable output (filterable with jq etc), hence the log keys like "`product.init`".